ci-operator/step-registry/cucushift/installer/check/cucushift-installer-check-commands.sh

```diff
@@ -26,4 +26,5 @@ run_command "oc get clusteroperators"
 run_command "oc get networks.config.openshift.io cluster -oyaml"
 run_command "oc get networks.operator.openshift.io cluster -oyaml"
 run_command "oc -n openshift-ingress-operator get ingresscontroller -oyaml"
+run_command "oc -n openshift-marketplace get catalogsources.operators.coreos.com"
```

Comment:
And I just checked `mirror-images-enable-qe-catalogsource-commands.sh`, the script has dependency on `mirror_registry_url`, while not each install workflow have the mirror registry, seem like not like what you stated in our meeting.

Reply:
And in the script, when it is a connected env, `create_catalog_sources` is referencing `MIRROR_PROXY_REGISTRY_QUAY`, but no such env is set in a connected env.

Reply:
That is because you are lucky to be working on aws, on aws, once bastion is created, `mirror_registry_url` will always be created. On other platform, we already hit some problems - https://github.com/openshift/release/pull/28087/files#r909405280.

Of course, my comment is not related with this PR. The PR looks good to me.

Reply:
Sorry for the late reply, I missed it. 
> Once mirror-images-enable-qe-catalogsource is enabled, this command will work, right?

Yes
> if we are developing a new install workflow, disable marketplace operator on purpose, will enabling mirror-images-enable-qe-catalogsource still make sense?

No, it won't work for now. Once `marketplace` disabled, the `openshift-marketplace` project will not be created as default. I can update the step if we needed to enaable the `QE CatalogSource` when `marketplace` disabled.

> the script has this line - MIRROR_REGISTRY_HOST=head -n 1 "${SHARED_DIR}/mirror_registry_url"`, but not each install workflow always create the file, need update?

Yes, I used it to distinguish the `disconnected` and `connected` envs, I will find a better solution, any suggestions?

> when it is a connected env, create_catalog_sources is referencing MIRROR_PROXY_REGISTRY_QUAY, but no such env is set in a connected env.

So, for connected env, which registry we're using? Thanks! I was told, it uses the same registry with disconnected env, but a different port.

Reply:
For connected env, I do not think mirror-registry is a MUST. E.g: a common ipi install, we never have a mirror registry there.